### ISSUE TITLE
Can't set Meslo font to Terminal:  The terminal only Supports monospace fonts

### ISSUE THREAD

**Issue by shikelong** [microsoft/vscode#88607]
Version: 1.41.1
Commit: 26076a4de974ead31f97692a0d32f90d735645c0
Date: 2019-12-18T14:57:51.166Z
Electron: 6.1.5
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Darwin x64 19.2.0
--------------

![Screen Shot 2020-01-14 at 21 17 33](https://user-images.githubusercontent.com/4499997/72347755-c9563d80-3713-11ea-9c25-167a8ba73eb8.png)
<img width="1246" alt="Screen Shot 2020-01-14 at 21 17 04" src="https://user-images.githubusercontent.com/4499997/72347767-cc512e00-3713-11ea-961d-4536abee9cce.png">

I installed the font family `Meslo` from https://github.com/powerline/fonts

Best Regards. 

Does this issue occur when all extensions are disabled?: No

**Comment by gjsjohnmurray** [comment]
[This comment](https://github.com/microsoft/vscode/issues/35681#issuecomment-371384431) by @Tyriar confirms what your notification tells you. The terminal only supports monospace fonts.

**Comment by Tyriar** [comment]
The problem is likely you have a space before the M in your font family:

![image](https://user-images.githubusercontent.com/2193314/72359793-c4f23c00-36a3-11ea-97f3-160830349f3e.png)

**Comment by gjsjohnmurray** [comment]
Great catch @Tyriar

**Comment by gjsjohnmurray** [comment]
Might be nice if the message was different when no font is found for the specified family, e.g.

![image](https://user-images.githubusercontent.com/6726799/72361716-df4f0880-36e9-11ea-8544-d0e38a9873b9.png)

The above screenshot comes from 1.42.0-insider which explains the extra hint about restarting if trying to use a newly-installed one.

**Comment by Tyriar** [comment]
@gjsjohnmurray yeah that's been suggested before, I just hand a string off to css/canvas though and the browser resolves the font based on that. I don't think it's possible to determine whether it resolved correctly or not.

**Comment by gjsjohnmurray** [comment]
@Tyriar maybe clearer then if the message was reworded as:

No matching monospace font was found. Check your spelling. Separate families with commas. Enclose family name in single quotes if it contains punctuation. If using a newly installed font, restart VS Code.

**Comment by Tyriar** [comment]
That's probably way too wordy for a notification, created https://github.com/microsoft/vscode/issues/88619 to add a read more link though.

**Comment by shikelong** [comment]
> The problem is likely you have a space before the M in your font family:
> 
> ![image](https://user-images.githubusercontent.com/2193314/72359793-c4f23c00-36a3-11ea-97f3-160830349f3e.png)

Thanks. I has trimed the leading space and the font can be used in vscode.
